e2e/env/env.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// This test sets singularity image specific environment variables and
+// verifies that they are properly set.
+
+package singularityenv
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/sylabs/singularity/e2e/internal/e2e"
+	"github.com/sylabs/singularity/internal/pkg/test"
+)
+
+type testingEnv struct {
+	// base env for running tests
+	CmdPath string `split_words:"true"`
+}
+
+var testenv testingEnv
+
+func singularityEnv(t *testing.T) {
+	// Singularity defines a path by default. See singularityware/singularity/etc/init.
```

Comment:
>// Singularity defines a path by default. See singularityware/singularity/etc/init.

Is this necessary? or should the url be updated to https://github.com/sylabs/singularity/blob/master/etc/init?

Reply:
@WestleyK Yeah, that should be updated. I put that in there before our repository migration I believe.